server/jukebox/jukebox.go

```diff
@@ -182,14 +184,20 @@ func (j *Jukebox) doUpdateSpeaker(su updateSpeaker) error {
 	j.Lock()
 	j.info = &strmInfo{}
 	j.info.strm = streamer.(beep.StreamSeekCloser)
+	if su.offset != 0 {
+		err = j.info.strm.Seek(format.SampleRate.N(time.Second * time.Duration(su.offset)))
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
		samples := format.SampleRate.N(time.Second * time.Duration(su.offset))
		if err := j.info.strm.Seek(samples); err != nil {
			return fmt.Errorf("seek stream: %w", err)
		}
```

how about an intermediary var what for doing the `if err := <>; err != nil {`